seerep_srv/seerep_core/src/core_project.cpp

```diff
@@ -173,6 +173,17 @@ std::vector<std::string> CoreProject::getFrames()
   return m_coreTfs->getFrames();
 }
 
+void CoreProject::reinitializeTFs()
+{
+  m_coreTfs->recreateBufferAndDatasets();
+  m_coreDatasets->recreateSpatialRt(seerep_core_msgs::Datatype::Image,
+                                    m_ioImage);
+  m_coreDatasets->recreateSpatialRt(seerep_core_msgs::Datatype::PointCloud,
+                                    m_ioPointCloud);
+  m_coreDatasets->recreateSpatialRt(seerep_core_msgs::Datatype::Point,
+                                    m_ioPoint);
```

Comment:
Can we do this in a loop based on `seerep_core_msgs::Datatype` ? To avoid missing it when adding another datatype